Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM zooniverse/panoptes-front-end:deps
+FROM node:8.12-alpine
+
+WORKDIR /src
+
+RUN apk update && apk upgrade && apk add --no-cache bash git openssh
```

Comment:
It's a minor thing, but there's no need to update/upgrade here. The `--no-cache` option loads the package index automatically.

Reply:
Good to know. I'm gonna pare this down to just adding git since bash isn't needed and ssh is a dep of git.